modules/ROOT/pages/upgrade-custom-policy-mule3.adoc

```diff
@@ -0,0 +1,35 @@
+= Upgrade Custom Policies for Mule 3
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In API Manager, you can upgrade the version of your custom policies for your Mule 3 applications.
+
+== Uploading a New Version of Your Custom Policy
```

Comment:
Should we point them to https://docs.mulesoft.com/api-manager/2.x/policy-mule3-creating-custom-policy#adding-the-custom-policy-to-api-manager ?